website/package.json

```diff
@@ -4,7 +4,8 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "next build && next export -o hosting/files/",
+    "release:web": "gh-pages -b website --dist .",
```

Comment:
Do we intend on having multiple release namespaces? Could we just do `release`? 

Shame we can't do `publish`.

Reply:
Yeah good call I can update this.